package-managers/dnf-qubes-hooks.py

```diff
@@ -20,9 +20,8 @@
 #
 
 from __future__ import absolute_import
-from distutils.version import LooseVersion
+
 import logging
-import dnf
```

Comment:
I think you meant removing `import dnf.const` instead. `import dnf` is used for `dnf.Plugin` base class.

Reply:
Yes.